modules/setup/tests/SetupControllerTest.php

```
@@ -10,50 +10,101 @@
+            if (!in_array($expectedAction, $actions, true)) {
+                $missingExpectedActions[] = $expectedAction;
+            }
         }
+
+        $unexpectedActionFound = in_array('missing_action_for_failure_path', $actions, true);
+
+        /* Assert */
+        self::assertSame(self::EXPECTED_ACTIONS, $actions);
+        self::assertSame([], $missingExpectedActions);
+        self::assertFalse($unexpectedActionFound);
     }
```

PHPMD (2.15.0)

[error] The method it_validates_all_actions_with_happy_and_failing_paths is not named in camelCase. (undefined)

(CamelCaseMethodName)